content/_data/changelogs/lts.yml

```
@@ -5622,6 +5622,98 @@
     message: |-
       Developer: Add support for plugins to use external SVG sprites in their icons.
 
+- version: "2.289.2"
+  date: 2021-06-30
+  changes:
+  - type: major bug
+    category: regression
+    pull: 5479
+    issue: 65585
+    authors:
+    - thomasgl-orange
+    - basil
+    references:
+      - issue: 65585
+      - pull: 5479
+      - pull: 5405
+    message: |-
+      Fix regressions in forms submissions: missing submit button value in form data (in HtmlUnit), double submit request (in HtmlUnit), and unwanted form validation in any browser (regression in 2.289.1).
```

This regression was not in the LTS line, it was reverted out of stable to be safe while we evaluated the fix. This change is more about preparing for an upcoming Firefox version

---

Thanks @timja for the clarification!  How about this as an alternative phrasing
```suggestion
      Prepare for form submission changes in future Firefox releases.
      Adapt form validation for all web browsers and test automation of form validation for HtmlUnit based tests.
```

---

Looks good

---

Fixed in e654e7582